suite-native/module-settings/src/components/EjectWallets/WalletRememberModeIconButton.tsx

```diff
@@ -58,6 +58,7 @@ export const WalletRememberModeIconButton = ({ device }: { device: TrezorDevice
                 onPress={handleEjectWallet}
                 colorScheme="tertiaryElevation1"
                 size="extraSmall"
+                testID={`@settings/eject-single-wallet/${device.features?.label}/${device.walletNumber ?? 0}`}
```

Comment:
Please don't use device label, better to use staticSessionId. Change it ideally in WalletRememberModeIconButton too. Device label is user input and can be sensitive.

Reply:
Fixed and changed the testId to `@settings/eject-single-wallet`